packages/name-controller/src/NameController.test.ts

```diff
@@ -115,26 +115,27 @@ describe('NameController', () => {
       const controller = new NameController({
         ...CONTROLLER_ARGS_MOCK,
         providers: [provider1],
-      });
-
-      controller.state.names = {
-        [NameType.ETHEREUM_ADDRESS]: {
-          [VALUE_MOCK]: {
-            [CHAIN_ID_MOCK]: {
-              name: null,
-              sourceId: null,
-              origin: null,
-              proposedNames: {
-                [SOURCE_ID_MOCK]: {
-                  proposedNames: [PROPOSED_NAME_MOCK, PROPOSED_NAME_2_MOCK],
-                  lastRequestTime: null,
-                  updateDelay: null,
+        state: {
```

Comment:
We have a few test files in which state was being modified directly, and I've had to fix them in order to make them pass. There are more that aren't addressed here, but those controllers are using BaseController v1 still, and we can fix them when we upgrade.